git.go

```diff
@@ -302,3 +334,97 @@ func ExecCmd(app string, command string) string {
 
 	return string(output)
 }
+
+func CommitBundle(app string) {
+	fmt.Println("COMMITING DEPLOYMENT BUNDLE TO GITHUB")
```

Comment:
is this something we want to keep? I think it would be nice to give the user some sort of visual feedback in terms of the steps that we're taking, but the format is definitely up to you

Reply:
Yeah I think i'll just move all the print statements to the provision function rather than scattered around in different functions